### ISSUE TITLE
Viafree "unable to retrieve the list of items"

### ISSUE THREAD

**Issue by lineslim** [retrospect-addon/plugin.video.retrospect#1438]
Since a week or so I get the message "unable to retrieve the list of items" when trying view the list of episodes for any program. I have updated the plugin via the Kodi menu. Very grateful if someone would take a look at this!

**Comment by basrieter** [comment]
As you did not include a DEBUG retrospect.log (and did not stick to the presented template), there is very little I can do right now. Please attach (don't copy paste) such a log:

1. Enable debug logging in Retrospect
2. Do whatever is not working
3. Upload the logfile with the built-in functionality of Retrospect
4. Post the link here.

**Comment by lineslim** [comment]
Sorry for not following the routine, thanks for pointing me in the right direction. Here is the logfile:
https://paste.kodi.tv/imayivixaw
Thanks

**Comment by basrieter** [comment]
This is not a retrospect.log, but a kodi.log. Please follow the steps above to provide the correct log.

**Comment by lineslim** [comment]
https://paste.kodi.tv/ivijutanuc
I hope its correct this time, Im learning ..:-)

**Comment by basrieter** [comment]
Yes, that is the one I needed. I see this line: 

```
20201022 22:53:33 - [ERROR   ] - urihandler.py        - 542  - GET failed with '403 Forbidden by HTTP PROXY' (0:00:00.130528) for http://playapi.mtgx.tv/v3/videos?format=6347&order=-airdate&type=program
```

Something is blocking your HTTPS requests.

**Comment by lineslim** [comment]
Thank you for solving my problem! I ruled out routers and internet connection & routers by connecting though my phones hotspot - same problem. Disabling the computers firewall also had no impact. But when I connected my vpn it was working, then disconnected and still working. I guess at some point there must have been an interruption of the vpn triggering a killswitch, which was then only partially unblocked leaving HTTPS blocked until connecting/disconnecting again.
